src/surfaces/filtersRail/filtersRail.jsx

```diff
@@ -210,6 +280,32 @@ function FiltersRail(props) {
                         </ScrollBar>
                     ) :
                         children}
+                    {!isEmpty(filterOptions) && !filterOptionsRequiredKeysMissing && (
+                        <div className={actionButtonsContainerClasses}>
+                            <div className={actionButtonsClasses}>
+                                {/* eslint-disable max-len */}
+                                {/* eslint-disable jsx-a11y/click-events-have-key-events, jsx-a11y/no-static-element-interactions, jsx-a11y/anchor-is-valid */}
```

Comment:
Do we have to suppress these?  Aren't both `<A>` and `<Button>` components compliant with all the accessibility rules?

Reply:
Yes, previously I was using HTML element `<a>` for that I was suppressing this error. With Implementation of <A>, it's not needed now. Thank you, I will remove it.

Reply:
Fixed. [Commit Link](https://github.com/saddlebackdev/react-cm-ui/pull/474/commits/52151f36990e262214ff2e97224c94097b3c1aec)